srcpkgs/python-brotli/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python-brotli'
+pkgname=python-brotli
+version=1.0.7
+revision=1
+wrksrc="Brotli-${version}"
+build_style="python-module"
+hostmakedepends="python-setuptools python-devel python3-setuptools python3-devel"
+short_desc="Python3 brotli module"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="Apache-2.0"
+homepage="https://pypi.org/project/Brotli/"
+distfiles="https://files.pythonhosted.org/packages/cd/9c/7955895f5672ecc85270244582c6b53ff95bb4c24bf77bd9271d42351635/Brotli-1.0.7.zip"
```

Comment:
```sh
distfiles=$PYPI_SITE/B/Brotli/Brotli-${version}.tar.gz
```

Reply:
Drop Python 2, too. It's EOL

Reply:
The brotli package isn't under the pypi site, I changed it now but I forgot to rebuild, it won't work with the new url

Reply:
OK, it should be:
```sh
distfiles=$PYPI_SITE/B/Brotli/Brotli-${version}.zip
```